code-challenges.js

```diff
@@ -58,12 +115,30 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+console.log(numberOfConnections.lastIndexOf(42))//Output 7
+console.log(numberOfConnections.lastIndexOf(10))//Output 8
+
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
 
+        //I had do do some reasearch, and figured that if i used the method .sort() alone, javascript thinks that i am trying to sort strings, and for numbers, it will sort them in the actual order of the numbers
+        //By using (a,b => b - a) i am telling it to compare functions in descending order which i believe is considered a comparison function
```

Comment:
Good research, though I would suggest to dive a little deeper to fully understand `.sort` and the comparison function you are passing as an argument!